pkg/server/server_sql.go

```diff
@@ -1298,6 +1298,8 @@ func (s *SQLServer) preStart(
 		return err
 	}
 
+	s.leaseMgr.SetRegionPrefix(regionPhysicalRep)
```

Comment:
regionPhysicalRep may be []byte{} and if it is, we should use enum.One instead. We discussed pushing this into GetLocalityRegionEnumPhysicalRepresentation, but I think that would be surprising behavior for the API. Instead, we could put the branch in server_sql. Alternatively we could defer initializing the leasing subsystem until a sql session is created and the lease could use the region from the session.

If we don't use enum.One, then we run into problems using sql to query leases that were created before the system database is configured as multi-region:

```
jeff@multiregion-8.6s4p.crdb.io:26257/defaultdb> select * from system.lease;
ERROR: internal error: unexpected error from the vectorized engine: could not find [] in enum "system.public.crdb_internal_region" representation PhysicalReps: [[64] [96] [128]]; LogicalReps: [gcp-asia-southeast1 gcp-europe-west1 gcp-us-central1] goroutine 12874 [running]:
```

Reply:
This is handled inside that call. It has the below logic. The value is initialized to enum.One.
```go
if len(val) > 0 {
		m.storage.regionPrefix.Store(val)
}
```